manual/guides/plots.md

```diff
@@ -101,3 +100,38 @@ plt.ylabel('Y Axis')
 plt.legend()
 plt.show()
 ```
+
+## 6. Inkscape
+
+[Inkscape](https://inkscape.org/about/) is a free vector graphics editor. 
+It's most notable feature is allowing for percise dimensions and units 
+for creating shapes, ensuring all figures are to scale. It is especailly
+helpful when you need to create graphics that can't easily be made
+automatically. Graphics are automatically saved as .svg files, but Inkscape 
+can also supports .pdf and .png files. While creating graphics, make sure to 
+follow the accessibility standards in regards to font, linetsyle, and color.
```

Comment:
Is there a link you could put here -- maybe to the writing guide where a new paragraph will eventually include an accessibility paragraph or something ?  (https://arfc.github.io/manual/guides/writing/)